draft-ietf-rtcweb-jsep.xml

```diff
@@ -2378,6 +2378,13 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             for bundled m= sections, as
             described in <xref target="I-D.ietf-mmusic-sdp-bundle-negotiation"/>,
             Section 8.1.
+            Note that if media m= sections are bundled into a
```

Comment:
I would make this its own para, as it's only loosely connected to a=bundle-only.